client/src/components/userlistVoting.tsx

```diff
@@ -0,0 +1,53 @@
+import { useUsers } from '../lib/hooks/users';
+import { getUserID } from '../lib/state/user';
+import { useVoted } from '../lib/hooks/voted';
+import styled from "styled-components"
+
+const ListStyleMe = styled.li`
+  list-style:none;
+  margin: 15px;
+  padding: 2px 32px 2px; 
+  font-family: 'Noto Sans JP';
+  width: 100px;
+  align-items: center;
+  border: 5px solid;
+  border-color: var(--primary-color);
+  background-color: var(--primary-color);
+`;
+
+const ListStylenotMe = styled.li`
+  list-style:none;
+  margin: 15px;
+  padding: 2px 32px 2px; 
+  font-family: 'Noto Sans JP';
+  width: 100px;
+  align-items: center;
+  border: 5px solid;
+  border: 5px solid;
+  border-color: var(--primary-color);
+  background-color: var(--background-color);
+`
+
+const UserList = () => {
+  const users = useUsers()
+  const userID = getUserID()
+  const voted = useVoted()
+
+  return (
+    <div>
+        <ul>
+          {users.map( item =>
+            item.id == userID?
```

Comment:
JavaScriptでは基本的に `==` (曖昧比較) は使いません `===` (厳密な比較) を使ってください
他の比較箇所も同様
```suggestion
            item.id === userID?
```